app/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="storage_csv_version">1</integer>
```

Comment:
NIT: This could as well be a string

Reply:
Sure. I don't like putting too much stuff into strings.xml, since then you have to mark it as being non-translatable, so integer seemed like a good choice - I hope that we won't need SemVer for the CSVs :-D

Reply:
Aand of course there's a bug in it, I access the resource ID, not the resource itself.. haven't worked with Android in a while.